spec/controllers/teams_controller_spec.rb

```diff
@@ -292,4 +292,36 @@
       }.to change(TeamUser, :count).by(-1)
     end
   end
+
+  describe 'POST #assign_membership_by_email' do
+    it 'succeeds when called as a team member' do
+      team = Team.create! valid_attributes
+      another_user = FactoryBot.create :user
+
+      team.members << subject.current_user
+
+      expect {
+        post :assign_membership_by_email, params: { id: team.id, email: another_user.email }
+      }.to change(team.members, :count).by(1)
+    end
+
+    it 'does not change members count when new user already member of team' do
+      team = Team.create! valid_attributes
+
+      team.members << subject.current_user
+
+      expect {
+        post :assign_membership_by_email, params: { id: team.id, email: subject.current_user.email }
+      }.to_not change(team.members, :count)
+    end
+
+    it 'does not succeed when email matches no member' do
+      team = Team.create! valid_attributes
+
+      team.members << subject.current_user
+
+      post :assign_membership_by_email, params: { id: team.id, email: nil }
```

Comment:
@bakoe It's probably not best practice to pass a `nil` email but this seemed more reasonable than making up some random string that won't match any email... But as always: open to discussion :-)

Reply:
even though i would have to try hard to care less, for consistency reasons: in existing tests for #23  we are facing a similar situation in which the string passed in a test is irrelevant; here we have chosen to just pass a sensible, random string instead of `nil`

Reply:
@tomkellog I somehow can't find that test you're talking about, could you add a link?